app/schema.graphql

```diff
@@ -644,6 +645,8 @@ type ExperimentRunAnnotation implements Node {
   metadata: JSON!
   startTime: DateTime!
   endTime: DateTime!
+  traceId: String
+  trace: Trace
```

Comment:
Nice, I guess for now I will need this (https://github.com/Arize-ai/phoenix/issues/3580) to fully qualify a URL for the trace. Not needed in this PR but will need this to create the linking.